project/scripts/dates.py

```diff
@@ -35,6 +35,11 @@ def get_current_date():
 
 def date_to_epoch(date):
     """
-    converts date object back into epoch
+    converts date object back into epoch (at exactly midnight utc on day of timestamp)
     """
-    return int(date.timestamp())
+    year, month, day = date.year, date.month, date.day
+    dt = datetime(year=year, month=month, day=day)
```

Comment:
why not just `dt = datetime(year=date.year, month=date.month, day=date.day)`?

Reply:
probably because I googled what I was doing in between writing those two lines....fixed now